src/main/resources/db/migration/V004__Schema.sql

```diff
@@ -0,0 +1,7 @@
+
+ALTER TABLE protected_characteristics ADD COLUMN IF NOT EXISTS dcn_number TEXT;
+
+ALTER TABLE protected_characteristics ADD COLUMN IF NOT EXISTS form_id TEXT;
+
+comment on column protected_characteristics.dcn_number is 'DCN Number for paper channel';
+comment on column protected_characteristics.form_id is 'Form type for paper channel';
```

Comment:
There is a slight issue here Manoj. Form type for Probate PCQ forms is EQ10, thats fine. However there isn't anything in the metadata which differentiates a Grant (PA1A, PA1P) from a Caveat (PA8A) which are seperate definitions in CCD.

GrantOfRepresentation CCD Case (PA1A - No Will, PA1P - Will)
Caveat CCD Case (PA8A - Caveat)